src/test/java/com/puppycrawl/tools/checkstyle/checks/metrics/ClassFanOutComplexityCheckTest.java

```diff
@@ -108,7 +110,11 @@ public void testExcludedPackagesCommonPackagesWithEndingDot() throws Exception {
 
     @Test
     public void testExcludedPackagesAllIgnored() throws Exception {
-        final String[] expected = CommonUtil.EMPTY_STRING_ARRAY;
```

Comment:
Shift in line numbers for violations is ok if necessary, we should not have a difference in the number of violations.

Reply:
actually while running `mvn clean verify` error came that there is a violation in that line. So i just changed the `// total: ok` to `// violation` comment. @nrmancuso